internal/databases/mongo/logical/storage_utils.go

```diff
@@ -0,0 +1,16 @@
+package logical
+
+import (
+	"github.com/wal-g/wal-g/internal/databases/mongo/common"
+	"github.com/wal-g/wal-g/internal/databases/mongo/models"
+	"github.com/wal-g/wal-g/pkg/storages/storage"
+)
+
+func DownloadSentinel(folder storage.Folder, backupName string) (*models.Backup, error) {
+	var sentinel models.Backup
+	err := common.DownloadSentinel(folder, backupName, &sentinel)
+	if sentinel.BackupName == "" {
```

Comment:
Better check the error and return it early to avoid some malformed struct being returned.
```
if err != nil {
    return nil, err
}
```